TrackletAlgorithm/VMStubMEMemoryCM.h

```diff
@@ -111,6 +111,14 @@ class VMStubMECM : public VMStubMECMBase<VMSMEType>
     return data_.range(kVMSMEBendMSB,kVMSMEBendLSB);
   }
 
+  /*bool isPSStub() const {
+    if(VMSMEType == BARRELPS) return true;
+    else if(VMSMEType == BARREL2S) return false;
+    else if(VMSMEType == DISK) return (getBend() & (1 << (VMStubMECMBase<VMSMEType>::kVMSMEBendSize-1))) == 0 && getBend() != 0; // Check highest 1 bits regardless of template type
+    static_assert(true, "Should not be possible!");
+  }*/
```

Comment:
@carriganm95 What's the purpose of adding this as a comment?

Reply:
sorry, I was removing this from a branch I copied from and forgot to delete. Should be fixed now